internal/engine/upper.go

```diff
@@ -384,7 +384,6 @@ func handleConfigsReloaded(
 			ms = &store.ManifestState{}
 		}
 		ms.LastManifestLoadError = nil
-		state.ManifestStates[m.ManifestName()].LastManifestLoadError = nil
```

Comment:
unless i'm missing something: we get `ms` from this map, and `ms` is a `*ManifestState`, so if we clear the error from `ms` in ln 386, there's no need to also clear the error from the thing in the map, because they're the same thing...?